plugins/grid-bookmark/src/GridBookmarkWidget/components/BookmarkGrid.tsx

```diff
@@ -66,98 +66,97 @@ const BookmarkGrid = observer(function ({
     setRowSelectionModel([])
   }, [bookmarkedRegions.length])
```

Comment:
random bug, not sure where it's happening:

If you have a set of 2 or more bookmarks, and delete the final one currently displayed in the grid, there is an app crash:

> Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.

This might have to do with the getters in the model being updated when this happens.

Reply:
nice catch. I think that maybe it can be fixed by deriving the rowSelectionModel only from the model.selectedBookmarks ([7b32387](https://github.com/GMOD/jbrowse-components/pull/3945/commits/7b32387241b1c14985eb3fe0964d7f4174682685))